docs/docs/advanced-concepts/enclave-edits.md

```diff
@@ -0,0 +1,20 @@
+---
+title: Enclave Edits
+sidebar_label: Enclave Edits
+---
+
+Enclave edits refers to Kurtosis' ability to make calls of [`kurtosis run`](../cli-reference/run.md) against an [enclave][enclaves] idempotent, meaning that the [plan](./plan.md) being submitted via `kurtosis run` is a declarative state of how the enclave should look and Kurtosis makes it so regardless of the current state of the enclave. In plain English, this means that Kurtosis will diff the plan being submitted via `kurtosis run` against what already exists in the enclave, and make only the changes necessary to get to the desired state.
+
+
+This has several uses:
+
+- **Speed:** when you're running a large [Starlark](./starlark.md) script or package and a step near the end has a bug, you don't want to start over from scratch with a fresh enclave and redo all the previous steps. Enclave edits allow you to simply fix your bug and resubmit, and Kurtosis will skip all the steps that have already been run. 
+- **Persistent environments:** persistent environments like shared Dev or Staging are instantiated at their start (Day 0), and then receive a constant updates into the future (Days 1+). In order for these environments to live in Kurtosis, Kurtosis needs to be able to handle Days 1+. Enclave edits allow this, as you to simply update your Starlark script and Kurtosis updates the environment in the enclave to match.
```

Comment:
updated `Eternal environments` to be `Persistent environments` since we used this terminology more often.